pom.xml

```diff
@@ -19,7 +19,9 @@
 		<published.container.port.production>9072</published.container.port.production>
 		<sandbox.db>false</sandbox.db>
 		<!-- Service properties -->
-		<pob.version>1.5</pob.version>
+		<openapi-generator-maven-plugin-version>5.4.0</openapi-generator-maven-plugin-version>
```

Comment:
It, doesn't work with newer version. Have to look into that. 5.4.0 was used in generated-clients as well. Maybe because of this...

Reply:
No, it's working. Change to 6.3.0!

You may have to refactor a bit (because of the null initialized objects in the generated classes).
Add Optional.ofNullable in some places and so on...

Reply:
You're so right. Fixed.